scripts/common.py

```diff
@@ -631,6 +632,26 @@ def wait_for_port_to_be_open(port_number):
         sys.exit(1)
 
 
+def wait_for_port_to_be_closed(port_number):
+    """Wait until the port is closed or
+    MAX_WAIT_TIME_FOR_PORT_TO_CLOSE_SECS seconds.
+
+    Args:
+        port_number: int. The port number to wait.
+
+    Returns:
+        bool. Whether the port closed in time.
+    """
+    waited_seconds = 0
+    while (is_port_open(port_number)
+           and waited_seconds < MAX_WAIT_TIME_FOR_PORT_TO_CLOSE_SECS):
+        time.sleep(1)
+        waited_seconds += 1
+    if is_port_open(port_number):
+        return False
+    return True
```

Comment:
`retrun not is_port_open(port_number)` [optional]

Reply:
done in ed1bcbd56